src/main/java/ch/uzh/ifi/hase/soprafs22/game/TurnInfo.java

```diff
@@ -0,0 +1,68 @@
+package ch.uzh.ifi.hase.soprafs22.game;
+
+import java.util.Optional;
+
+/**
+ * Holds data related to a turn.
+ */
+public class TurnInfo {
+    private final int turn;
+    private final long playerId;
+
+    private TurnInfo(int turn, long playerId) {
+        this.turn = turn;
+        this.playerId = playerId;
+    }
+
+    /**
+     * Get the turn number.
+     * @return the turn number.
+     */
+    public int getTurn() {
+        return turn;
+    }
+
+    /**
+     * Get the player Id whose turn it is.
+     * @return the id of the player whose turn it is.
+     */
+    public long getPlayerId() {
+        return playerId;
+    }
+
+    /**
+     * @return a new builder for TurnInfo.
+     */
+    public static Builder newBuilder() {
```

Comment:
why do you use a builder here?
couldn't we just make the constructor public and use regular getters/setters?

I'm not familiar with the Builder here ;)